[2025-08-21 15:50:45] Deploy Connector: === Starting Connector Deployment: Deploying postgres-cdc-connector ===
[2025-08-21 15:50:45] Deploy Connector: Validating prerequisites...
[2025-08-21 15:50:45] Deploy Connector: SUCCESS Prerequisites validated
[2025-08-21 15:50:45] Deploy Connector: Checking Kafka Connect service status...
NAME                             READY   STATUS    RESTARTS   AGE
kafka-connect-7b7d8885c5-k8g2m   1/1     Running   0          82s
[2025-08-21 15:50:45] Deploy Connector: Waiting for Kafka Connect to be ready...
[2025-08-21 15:50:45] Deploy Connector: SUCCESS Kafka Connect is ready
[2025-08-21 15:50:46] Deploy Connector: Checking if connector already exists...
[2025-08-21 15:50:46] Deploy Connector: SUCCESS No existing connector found
[2025-08-21 15:50:46] Deploy Connector: Deploying connector...
[2025-08-21 15:50:47] Deploy Connector: SUCCESS Connector deployed successfully
[2025-08-21 15:50:48] Deploy Connector: Checking connector status...
{
  "name": "postgres-cdc-connector",
  "connector": {
    "state": "RUNNING",
    "worker_id": "kafka-connect:8083"
  },
  "tasks": [
    {
      "id": 0,
      "state": "RUNNING",
      "worker_id": "kafka-connect:8083"
    }
  ],
  "type": "source"
}
[2025-08-21 15:50:53] Deploy Connector: SUCCESS Connector is in RUNNING state
[2025-08-21 15:50:53] Deploy Connector: Listing all connectors...
[
  "postgres-cdc-connector"
]
[2025-08-21 15:50:54] Deploy Connector: SUCCESS Connector Deployment completed successfully
[2025-08-21 15:50:54] Deploy Connector: === Starting Connector Deployment: Deploying s3-sink-connector ===
[2025-08-21 15:50:54] Deploy Connector: Validating prerequisites...
[2025-08-21 15:50:54] Deploy Connector: SUCCESS Prerequisites validated
[2025-08-21 15:50:54] Deploy Connector: Checking Kafka Connect service status...
NAME                             READY   STATUS    RESTARTS   AGE
kafka-connect-7b7d8885c5-k8g2m   1/1     Running   0          91s
[2025-08-21 15:50:54] Deploy Connector: Waiting for Kafka Connect to be ready...
[2025-08-21 15:50:54] Deploy Connector: SUCCESS Kafka Connect is ready
[2025-08-21 15:50:55] Deploy Connector: Checking if connector already exists...
[2025-08-21 15:50:55] Deploy Connector: SUCCESS No existing connector found
[2025-08-21 15:50:55] Deploy Connector: Deploying connector...
[2025-08-21 15:50:56] Deploy Connector: SUCCESS Connector deployed successfully
[2025-08-21 15:50:56] Deploy Connector: Checking connector status...
{
  "name": "s3-sink-connector",
  "connector": {
    "state": "RUNNING",
    "worker_id": "kafka-connect:8083"
  },
  "tasks": [
    {
      "id": 0,
      "state": "FAILED",
      "worker_id": "kafka-connect:8083",
      "trace": "org.apache.kafka.connect.errors.ConnectException: com.amazonaws.services.s3.model.AmazonS3Exception: <YOUR-AKID>/YYYYMMDD/REGION/SERVICE/aws4_request\". (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed; Request ID: ENGDQDTRVAY7ZH3F; S3 Extended Request ID: kkFYZEoKSuHrSwyL5xfv1EsW93QoNNF+Q2X+udkpjuD5RZnVvorl6q8CDZGnG/JgalgbRLHtziU=; Proxy: null), S3 Extended Request ID: kkFYZEoKSuHrSwyL5xfv1EsW93QoNNF+Q2X+udkpjuD5RZnVvorl6q8CDZGnG/JgalgbRLHtziU=\n\tat io.confluent.connect.s3.S3SinkTask.start(S3SinkTask.java:142)\n\tat org.apache.kafka.connect.runtime.WorkerSinkTask.initializeAndStart(WorkerSinkTask.java:315)\n\tat org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:200)\n\tat org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:257)\n\tat org.apache.kafka.connect.runtime.isolation.Plugins.lambda$withClassLoader$1(Plugins.java:181)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: com.amazonaws.services.s3.model.AmazonS3Exception: <YOUR-AKID>/YYYYMMDD/REGION/SERVICE/aws4_request\". (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed; Request ID: ENGDQDTRVAY7ZH3F; S3 Extended Request ID: kkFYZEoKSuHrSwyL5xfv1EsW93QoNNF+Q2X+udkpjuD5RZnVvorl6q8CDZGnG/JgalgbRLHtziU=; Proxy: null), S3 Extended Request ID: kkFYZEoKSuHrSwyL5xfv1EsW93QoNNF+Q2X+udkpjuD5RZnVvorl6q8CDZGnG/JgalgbRLHtziU=\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1879)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleServiceErrorResponse(AmazonHttpClient.java:1418)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1387)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1157)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:814)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:781)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:755)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:715)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:697)\n\tat com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:561)\n\tat com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:541)\n\tat com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5456)\n\tat com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5403)\n\tat com.amazonaws.services.s3.AmazonS3Client.getAcl(AmazonS3Client.java:4062)\n\tat com.amazonaws.services.s3.AmazonS3Client.getBucketAcl(AmazonS3Client.java:1278)\n\tat com.amazonaws.services.s3.AmazonS3Client.getBucketAcl(AmazonS3Client.java:1268)\n\tat com.amazonaws.services.s3.AmazonS3Client.doesBucketExistV2(AmazonS3Client.java:1406)\n\tat io.confluent.connect.s3.storage.S3Storage.bucketExists(S3Storage.java:186)\n\tat io.confluent.connect.s3.S3SinkTask.start(S3SinkTask.java:114)\n\t... 9 more\n"
    },
    {
      "id": 1,
      "state": "FAILED",
      "worker_id": "kafka-connect:8083",
      "trace": "org.apache.kafka.connect.errors.ConnectException: com.amazonaws.services.s3.model.AmazonS3Exception: <YOUR-AKID>/YYYYMMDD/REGION/SERVICE/aws4_request\". (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed; Request ID: ENGE4CAC09Z2KAQS; S3 Extended Request ID: nAxANV49R331L4t9dnXtTJNW7UhKCbkmkrJ8aUwT7woFWCfgbwrXYl6aTJAwUkF+W8jgNo75MXE=; Proxy: null), S3 Extended Request ID: nAxANV49R331L4t9dnXtTJNW7UhKCbkmkrJ8aUwT7woFWCfgbwrXYl6aTJAwUkF+W8jgNo75MXE=\n\tat io.confluent.connect.s3.S3SinkTask.start(S3SinkTask.java:142)\n\tat org.apache.kafka.connect.runtime.WorkerSinkTask.initializeAndStart(WorkerSinkTask.java:315)\n\tat org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:200)\n\tat org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:257)\n\tat org.apache.kafka.connect.runtime.isolation.Plugins.lambda$withClassLoader$1(Plugins.java:181)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: com.amazonaws.services.s3.model.AmazonS3Exception: <YOUR-AKID>/YYYYMMDD/REGION/SERVICE/aws4_request\". (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed; Request ID: ENGE4CAC09Z2KAQS; S3 Extended Request ID: nAxANV49R331L4t9dnXtTJNW7UhKCbkmkrJ8aUwT7woFWCfgbwrXYl6aTJAwUkF+W8jgNo75MXE=; Proxy: null), S3 Extended Request ID: nAxANV49R331L4t9dnXtTJNW7UhKCbkmkrJ8aUwT7woFWCfgbwrXYl6aTJAwUkF+W8jgNo75MXE=\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1879)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleServiceErrorResponse(AmazonHttpClient.java:1418)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1387)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1157)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:814)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:781)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:755)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:715)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:697)\n\tat com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:561)\n\tat com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:541)\n\tat com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5456)\n\tat com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5403)\n\tat com.amazonaws.services.s3.AmazonS3Client.getAcl(AmazonS3Client.java:4062)\n\tat com.amazonaws.services.s3.AmazonS3Client.getBucketAcl(AmazonS3Client.java:1278)\n\tat com.amazonaws.services.s3.AmazonS3Client.getBucketAcl(AmazonS3Client.java:1268)\n\tat com.amazonaws.services.s3.AmazonS3Client.doesBucketExistV2(AmazonS3Client.java:1406)\n\tat io.confluent.connect.s3.storage.S3Storage.bucketExists(S3Storage.java:186)\n\tat io.confluent.connect.s3.S3SinkTask.start(S3SinkTask.java:114)\n\t... 9 more\n"
    },
    {
      "id": 2,
      "state": "FAILED",
      "worker_id": "kafka-connect:8083",
      "trace": "org.apache.kafka.connect.errors.ConnectException: com.amazonaws.services.s3.model.AmazonS3Exception: <YOUR-AKID>/YYYYMMDD/REGION/SERVICE/aws4_request\". (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed; Request ID: ENGA2H2FGEY1419A; S3 Extended Request ID: JYS6QwQW44hlymo3KyP1PSnpxsNgXGpmXoaFs7cIk+6vVMxQwEvsqqT0hNtux9umVmtJJ5qq0iY=; Proxy: null), S3 Extended Request ID: JYS6QwQW44hlymo3KyP1PSnpxsNgXGpmXoaFs7cIk+6vVMxQwEvsqqT0hNtux9umVmtJJ5qq0iY=\n\tat io.confluent.connect.s3.S3SinkTask.start(S3SinkTask.java:142)\n\tat org.apache.kafka.connect.runtime.WorkerSinkTask.initializeAndStart(WorkerSinkTask.java:315)\n\tat org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:200)\n\tat org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:257)\n\tat org.apache.kafka.connect.runtime.isolation.Plugins.lambda$withClassLoader$1(Plugins.java:181)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: com.amazonaws.services.s3.model.AmazonS3Exception: <YOUR-AKID>/YYYYMMDD/REGION/SERVICE/aws4_request\". (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed; Request ID: ENGA2H2FGEY1419A; S3 Extended Request ID: JYS6QwQW44hlymo3KyP1PSnpxsNgXGpmXoaFs7cIk+6vVMxQwEvsqqT0hNtux9umVmtJJ5qq0iY=; Proxy: null), S3 Extended Request ID: JYS6QwQW44hlymo3KyP1PSnpxsNgXGpmXoaFs7cIk+6vVMxQwEvsqqT0hNtux9umVmtJJ5qq0iY=\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1879)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleServiceErrorResponse(AmazonHttpClient.java:1418)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1387)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1157)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:814)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:781)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:755)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:715)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:697)\n\tat com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:561)\n\tat com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:541)\n\tat com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5456)\n\tat com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5403)\n\tat com.amazonaws.services.s3.AmazonS3Client.getAcl(AmazonS3Client.java:4062)\n\tat com.amazonaws.services.s3.AmazonS3Client.getBucketAcl(AmazonS3Client.java:1278)\n\tat com.amazonaws.services.s3.AmazonS3Client.getBucketAcl(AmazonS3Client.java:1268)\n\tat com.amazonaws.services.s3.AmazonS3Client.doesBucketExistV2(AmazonS3Client.java:1406)\n\tat io.confluent.connect.s3.storage.S3Storage.bucketExists(S3Storage.java:186)\n\tat io.confluent.connect.s3.S3SinkTask.start(S3SinkTask.java:114)\n\t... 9 more\n"
    },
    {
      "id": 3,
      "state": "FAILED",
      "worker_id": "kafka-connect:8083",
      "trace": "org.apache.kafka.connect.errors.ConnectException: com.amazonaws.services.s3.model.AmazonS3Exception: <YOUR-AKID>/YYYYMMDD/REGION/SERVICE/aws4_request\". (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed; Request ID: ENG0SW23ZDW0F3HT; S3 Extended Request ID: SdVmFFe1p7+J3OAPV10KKSI8zV7dp1J2cyHh70fahmiKd8OyTxXWx7AMg/ly/eZbOUA1kxiYOMc=; Proxy: null), S3 Extended Request ID: SdVmFFe1p7+J3OAPV10KKSI8zV7dp1J2cyHh70fahmiKd8OyTxXWx7AMg/ly/eZbOUA1kxiYOMc=\n\tat io.confluent.connect.s3.S3SinkTask.start(S3SinkTask.java:142)\n\tat org.apache.kafka.connect.runtime.WorkerSinkTask.initializeAndStart(WorkerSinkTask.java:315)\n\tat org.apache.kafka.connect.runtime.WorkerTask.doRun(WorkerTask.java:200)\n\tat org.apache.kafka.connect.runtime.WorkerTask.run(WorkerTask.java:257)\n\tat org.apache.kafka.connect.runtime.isolation.Plugins.lambda$withClassLoader$1(Plugins.java:181)\n\tat java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)\n\tat java.base/java.lang.Thread.run(Thread.java:829)\nCaused by: com.amazonaws.services.s3.model.AmazonS3Exception: <YOUR-AKID>/YYYYMMDD/REGION/SERVICE/aws4_request\". (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed; Request ID: ENG0SW23ZDW0F3HT; S3 Extended Request ID: SdVmFFe1p7+J3OAPV10KKSI8zV7dp1J2cyHh70fahmiKd8OyTxXWx7AMg/ly/eZbOUA1kxiYOMc=; Proxy: null), S3 Extended Request ID: SdVmFFe1p7+J3OAPV10KKSI8zV7dp1J2cyHh70fahmiKd8OyTxXWx7AMg/ly/eZbOUA1kxiYOMc=\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleErrorResponse(AmazonHttpClient.java:1879)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.handleServiceErrorResponse(AmazonHttpClient.java:1418)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1387)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1157)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:814)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:781)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:755)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:715)\n\tat com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:697)\n\tat com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:561)\n\tat com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:541)\n\tat com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5456)\n\tat com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5403)\n\tat com.amazonaws.services.s3.AmazonS3Client.getAcl(AmazonS3Client.java:4062)\n\tat com.amazonaws.services.s3.AmazonS3Client.getBucketAcl(AmazonS3Client.java:1278)\n\tat com.amazonaws.services.s3.AmazonS3Client.getBucketAcl(AmazonS3Client.java:1268)\n\tat com.amazonaws.services.s3.AmazonS3Client.doesBucketExistV2(AmazonS3Client.java:1406)\n\tat io.confluent.connect.s3.storage.S3Storage.bucketExists(S3Storage.java:186)\n\tat io.confluent.connect.s3.S3SinkTask.start(S3SinkTask.java:114)\n\t... 9 more\n"
    }
  ],
  "type": "sink"
}
[2025-08-21 15:51:02] Deploy Connector: SUCCESS Connector is in RUNNING state
[2025-08-21 15:51:02] Deploy Connector: Listing all connectors...
[
  "s3-sink-connector",
  "postgres-cdc-connector"
]
[2025-08-21 15:51:02] Deploy Connector: SUCCESS Connector Deployment completed successfully
